src/network/networkd-dhcp6.c

```diff
@@ -137,7 +137,7 @@ static int dhcp6_pd_remove_old(Link *link, bool force) {
         assert(link);
         assert(link->manager);
 
-        if (!force && (link->dhcp6_pd_address_messages != 0 || link->dhcp6_pd_route_configured != 0))
+        if (!force && (link->dhcp6_pd_address_messages != 0 || link->dhcp6_pd_route_messages != 0))
```

Comment:
"fix typo" doesn't really do just to this patch ;) I'd say "fix condition check" instead.

Reply:
The commit message is updated.